plugins/backend/catalog-backend-module-okta/src/providers/types.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GroupEntity, UserEntity } from '@backstage/catalog-model';
+import type {
+  Group as OktaGroup,
+  User as OktaUser,
+} from '@okta/okta-sdk-nodejs';
+import { GroupNamingStrategy } from './groupNamingStrategies';
+import { UserNamingStrategy } from './userNamingStrategies';
+
+export type OktaGroupEntityTransformer = (
+  group: OktaGroup,
+  namingStrategy: GroupNamingStrategy,
+  parentGroup: OktaGroup | undefined,
```

Comment:
I think we could make parentGroup optional so we dont need to write out the explicit undefined here
```suggestion
  parentGroup?: OktaGroup,
```

Reply:
Good catch! Done